sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/OrganizationDetails.java

```diff
@@ -23,7 +23,7 @@
      * Details of the organization administrator.
      */
     @JsonProperty(value = "admin_details")
-    private List<Administrator> adminDetails;
```

Comment:
Should the name of this class be singular `OrganizationDetail` ?
I remember making this change for one of the classes in core.
@JonathanGiles

Reply:
This is a standard KV API review change for all languages.

Reply:
It's ok as-is, because it represents a collection of details, not a single detail.